docker-compose.dev-build.yml

```diff
@@ -0,0 +1,18 @@
+version: "3.9"
+
+services:
+
+  tile_cache:
+    image: registry.cevi.tools/cevi/awtg_tile_caching:dev
```

Comment:
`awtg` -> Automatic Walktime Table G?

Reply:
... generator, but I will change it to awt_*